lib/bolt/util.rb

```diff
@@ -273,6 +273,11 @@ def windows?
         !!File::ALT_SEPARATOR
       end
 
+      # Returns true if running in PowerShell.
+      def powershell?
+        !!ENV['PSModulePath']
```

Comment:
@jpogran Does this seem like a reasonable way to know if we're in powershell vs. CMD or *sh?

Reply:
This is actually his suggestion!